packages/@okta/vuepress-site/architecture-center/case-studies/companyx/index.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: Landing
+title: User Migration for Company X
+---
+
+# User Migration for Company X
+
+Learn how Company X applied the directory coexistence reference architecture when planning their user migration strategy and the benefits they saw.
+
+## Company Overview
+
+Company X is a Global 2000 company with 200,000 team members worldwide. It uses over 250 SaaS apps, 500 on-premises apps, and 400 cloud-native apps backed by five legacy Identity and Access Management (IAM) systems and multiple Identity Providers (IdPs).
+
+## Related Reference Architecture
+
+<figure class="main-card">
+  <img
+    class="main-card__image"
+    src="/img/architecture/thumbs/usermgmt.png"
+    alt="UserManagement"
+  >
+  <figcaption class="main-card__body">
+    <h2 class="main-card__title">
+      <a
+        class="main-card__main-link"
+        href="/architecture-center/reference-architectures/directory-coexistence"
+      >
+        Directory Coexistence
+      </a>
+    </h2>
+    <div class="main-card__wrapper">
+      <span class="main-card__tag">
+        Architecture paper
+      </span>
+      <span class="main-card__tag">
+        User migration
+      </span>
+      <span class="main-card__tag">
+        Universal Directory
+      </span>
+    </div>
+    <p class="main-card__text">
+      Minimize downtime while migrating user, group, and device profiles to Okta Universal Directory by keeping both source and target directories active during the process. <a class="main-card__link" href="/architecture-center/reference-architectures/directory-coexistence/">Learn more</a>
+    </p>
+  </figcaption>
+</figure>
+
+## The Problem: Too many legacy IAM systems spoil the user administration broth
```

Comment:
Avoid using colon in the heading. Maybe move some parts of the heading to the body text?
Avoid jargons too

```suggestion
## The problem

Managing users is a challenge with too many legacy IAM systems.
```

Reply:
Why no colon in headings?